api/src/main/java/com/datastrato/gravitino/authorization/Privilege.java

```diff
@@ -81,34 +81,28 @@ enum Name {
     WRITE_TOPIC(0L, 1L << 18),
     /** The privilege to read a topic. */
     READ_TOPIC(0L, 1L << 19),
-    /** The privilege to create a metalake. */
-    CREATE_METALAKE(0L, 1L << 20),
-    /** The privilege to manage a metalake, including drop and alter a metalake. */
-    MANAGE_METALAKE(0L, 1L << 21),
-    /** The privilege to use a metalake, the user can load the information of the metalake. */
-    USE_METALAKE(0L, 1L << 22),
     /** The privilege to add a user */
-    ADD_USER(0L, 1L << 23),
+    ADD_USER(0L, 1L << 20),
```

Comment:
You'd better try to avoid changing the value of `highBits` and `lowBits` of existing Enum.

Reply:
It's ok to change this. Because we don't persist them in the underlying storage. We only store the privilege name.